docs/data-sources/cce_addon.md

```diff
@@ -0,0 +1,37 @@
+---
+subcategory: "Cloud Container Engine (CCE)"
+---
+
+# huaweicloud\_cce\_addon
+
+To get the specified addon in a cluster.
+
+## Example Usage
+
+```hcl
+variable "cluster_id" { }
+variable "tempalte_name" { }
+
+data "huaweicloud_cce_addon" "addon" {
+  cluster_id = var.cluster_id
+  name       = var.tempalte_name
+}
+```
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) The region in which to create the cce addon resource.
+ If omitted, the provider-level region will be used.
+* `cluster_id` - (Required, String) Specifies the ID of the cluster.
+* `addon_id` - (Required, String) Specifies the ID of the addon.
```

Comment:
`addon_id` is required in docs while it is optional is the schema.

Reply:
fixed